### PR TITLE
Support Nunjucks in Markdown files in docs

### PR DESCRIPTION
**Pull request by lfdebrux**

### CODE REVIEW

**Comment by lfdebrux** [comment]
I compared the output HTML of the page `/docs/install/getting-started` for both this PR and PR #121, and the output was more or less the same, excepting some differences in whitespace.

File: lib/render.js [hunk 1]
```diff
@@ -43,3 +48,37 @@ exports.markdownEngine = (nunjucksEnv) => {
     })
   }
 }
+
+// Use with app.engine('.njk', nunjucksAndMarkdownEngine(nunjucksEnv))
+// Express can't handle with an engine with multiple file extensions
```
Comment (joelanman) [inline lib/render.js]: typo?
>Express can't handle with an engine

File: lib/render.js [hunk 2]
```diff
@@ -43,3 +48,37 @@ exports.markdownEngine = (nunjucksEnv) => {
...
+exports.nunjucksAndMarkdownEngine = function (nunjucksEnv) {
+  return (filePath, options, callback) => {
```
Comment (joelanman) [inline lib/render.js:55]: not a strong opinion but might it be easier to read as 3 functions? To do
 - matter
 - nunjucks
 - markdown

Comment (lfdebrux) [reply lib/render.js:55]: Yeah, I tried that initially, but I found it was harder to follow the logic.

File: lib/render.js (hunk 1 above)
Comment (lfdebrux) [reply lib/render.js]: Thanks, fixed.

**Review by joelanman — APPROVED** [review]

**Pull request merged**